augur/market_info.py

```diff
@@ -219,7 +220,7 @@ def category(self):
   def tags(self):
     '''Names with which the Market has been tagged.
 
-    Returns list<(str)> or None
+    Returns list<(str or None)>
```

Comment:
For lists use the convention ```list[str]```.

Also determine if this is a ```list[str or None]``` or a ```list[str] or None```. The two are very different. If it's a ```list[str] or None``` then eliminate the ```None``` and instead this function will return an empty list.